pages/vision.html

```diff
@@ -5,6 +5,36 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Our Vision</title>
+		<link rel="stylesheet" href="../styles/vision_style.css">
 	</head>
-	<body></body>
+	<body>
+		<header>
```

Comment:
Change this header tag for my header tag in index.html file

Reply:
done